design/apidsl/security.go

```diff
@@ -0,0 +1,308 @@
+package apidsl
+
+import (
+	"github.com/goadesign/goa/design"
+	"github.com/goadesign/goa/dslengine"
+)
+
+// Security defines an authentication method required to access a goa
+// Action.  When defined on a Resource, it applies to all Actions,
+// unless overriden by individual actions.  When defined at the API
+// level, it will apply to all resources by default, following the
+// same logic.
+//
+// The method name refers to what you have passed to the different
```

Comment:
Yes..

We want that in the first merge ? Seems like a non breaking change / feature we can add right after.